woocommerce-gateway-ebanx/woocommerce-gateway-ebanx.php

```diff
@@ -317,7 +288,11 @@ public function check_merchant_api_keys()
 				$api_url = 'https://api.ebanx.com';
 
 				$message = sprintf('Could not connect to EBANX servers. Please check if your server can reach our API (<a href="%1$s">%1$s</a>) and your integrations keys are correct.', $api_url);
-				$this->add_admin_notice('connection_error', 'error', $message);
+				$notice = new WC_EBANX_Notices_Notice();
```

Comment:
Can you avoid multiple calls of `WC_EBANX_Notices_Notice` using the property `$this->notice`?